docs/quickstart/create-and-publish-a-package-using-visual-studio-net-framework.md

```diff
@@ -1,162 +1,176 @@
+1. (Optional) To see and edit the properties directly, open the *Properties/AssemblyInfo.cs* file in the project by selecting select **Project** > **Edit Project File**.
 
-1. When the properties are set, set the project configuration to **Release** and rebuild the project to generate the updated DLL.
+1. After you've set these properties, set the **Active solution configuration** in **Build** > **Configuration Manager** to **Release** and rebuild the project to generate the updated DLL.
 
 ## Generate the initial manifest
 
-With a DLL in hand and project properties set, you now use the `nuget spec` command to generate an initial `.nuspec` file from the project. This step includes the relevant replacement tokens to draw information from the project file.
+After you've set the project properties and created the DLL, you can now generate an initial *.nuspec* file from the project. This step includes the relevant replacement tokens to draw information from the project file.
+
+Run `nuget spec` only once to generate the initial manifest. If you update the package, either change values in your project, or edit the manifest directly:
 
-You run `nuget spec` only once to generate the initial manifest. When updating the package, you either change values in your project or edit the manifest directly.
+1. With your project open in **Solution Explorer**, open a command prompt by selecting **Tools** > **Command Line** >  **Developer Command Prompt**.
 
-1. Open a command prompt and navigate to the project folder containing `AppLogger.csproj` file.
+   The command prompt opens in your project directory where the `AppLogger.csproj` file is located.
 
-1. Run the following command: `nuget spec AppLogger.csproj`. By specifying a project, NuGet creates a manifest that matches the name of the project, in this case `AppLogger.nuspec`. It also include replacement tokens in the manifest.
+1. Run the following command: `nuget spec AppLogger.csproj`.
 
-1. Open `AppLogger.nuspec` in a text editor to examine its contents, which should appear as follows:
+   NuGet creates a manifest that matches the name of the project, in this case `AppLogger.nuspec`. It also includes replacement tokens in the manifest.
+
+1. Open `AppLogger.nuspec` in a text editor to examine its contents, which will be similar to the following code:
 
     ```xml
     <?xml version="1.0"?>
     <package >
       <metadata>
         <id>Package</id>
         <version>1.0.0</version>
-        <authors>YourUsername</authors>
-        <owners>YourUsername</owners>
+        <authors>Your username</authors>
+        <owners>Your username</owners>
         <license type="expression">MIT</license>
-        <projectUrl>http://PROJECT_URL_HERE_OR_DELETE_THIS_LINE</projectUrl>
-        <iconUrl>http://ICON_URL_HERE_OR_DELETE_THIS_LINE</iconUrl>
+        <projectUrl>http://ENTER_PROJECT_URL_HERE_OR_DELETE_THIS_LINE</projectUrl>
+        <iconUrl>http://ENTER_ICON_URL_HERE_OR_DELETE_THIS_LINE</iconUrl>
```

Comment:
I suggest not changing these two URLs.  `http://PROJECT_URL_HERE_OR_DELETE_THIS_LINE` is what nuget.exe adds when you run `nuget spec`, and more importantly, `nuget pack` will detect that URL and write an error to the console, telling the customer they need to change it.  Since `http://ENTER_PROJECT_URL_HERE_OR_DELETE_THIS_LINE` is different, nuget will allow the value and will not error or warn.

Additionally, icon URLs are deprecated now, we want customers to migrate to embedded icons. I think, but I'm not 100% sure, that `nuget spec` no longer creates the `<iconUrl>` element. I think it should be removed from this doc.